aaudio/echo/src/main/cpp/echo_audio_engine.cc

```diff
@@ -123,11 +123,13 @@ void EchoAudioEngine::stopStreams() {
   if (recordingStream_ != nullptr) {
```

Comment:
The destructor also needs to either check if the pointers are null before closing,
or just call stopStreams(),
or assert that the pointers are null.